apps/wing/src/commands/run.ts

```diff
@@ -1,21 +1,23 @@
-import { readdirSync  } from "fs";
+import { readdirSync, existsSync  } from "fs";
 import { debug } from "debug";
 import { resolve } from "path";
 import open = require("open");
 import { normalPath } from "../util";
 
 export async function run(simfile?: string) {
-    const wingFiles = readdirSync('.').filter(item => item.endsWith('.w'));
     if (!simfile) {
+        const wingFiles = readdirSync('.').filter(item => item.endsWith('.w'));
         if (wingFiles.length !== 1) {
             throw new Error('Please specify which file you want to run');
         }
         simfile = wingFiles[0];
-    } else if (!wingFiles.includes(simfile)) {
+    }
+    
+    if (!existsSync(simfile)) {
         throw new Error(simfile + " doesn't exist");
     }
 
     simfile = normalPath(resolve(simfile));
```

Comment:
don't you need to normalize the path before checking `!existsSync()`? I mean, will `existsSync()` work cross platform if it runs on simFile path that has not been normalized?

Reply:
existsSync() uses platform-specific logic, so yeah it'll work without normalization